Source/WebCore/Modules/cache/DOMCacheStorage.cpp

```diff
@@ -222,23 +222,26 @@ void DOMCacheStorage::open(const String& name, DOMPromiseDeferred<IDLInterface<D
 
 void DOMCacheStorage::doOpen(const String& name, DOMPromiseDeferred<IDLInterface<DOMCache>>&& promise)
 {
+    RefPtr context = scriptExecutionContext();
+    if (!context)
```

Comment:
what happens to the DOMPromise here? does it need to be resolved/rejected?

Reply:
Right, it is consistent to reject it.